index.html

```diff
@@ -1222,19 +1222,36 @@ <h1>Common Deployment Patterns</h1>
     </p>
 
     <section id="telemetry">
-      <h3>Telemetry</h3>    
-      <section class="ednote" title="TODO">
-      <h3>TODO</h3>
-      Create a description and a diagram for a telemetry scenario.
-      </section>
+      <h3>Telemetry</h3>  
+      <p>
+        Telemetry is the monitoring of remove devices, which implies automatic transmission to, and processing
+        of measurements and other data on a <a>Consumer</a>. 
```

Comment:
Consumer: Since the definition of WoT entities (Thing/Intermediary/Consumer) is given later in Chapter 6, so what about to use a generic term (e.g. "data collector") rather than a WoT-specific term here?

Reply:
@k-toumura If we would use "data collector" we would have to define it in the terminology.
A "data collector" sounds like it is only storing the data. Typically the consumer does processing, applies rules, ...

Reply:
We could use "data consumer" if it makes the text clearer, what do you think?